crypto/x509v3/v3_cpols.c

```diff
@@ -291,13 +293,53 @@ static POLICYINFO *policy_section(X509V3_CTX *ctx,
 
 }
 
+static int displaytext_get_tag_len(const char *tagstr)
+{
+    char *colon;
+    colon = strchr(tagstr, ':');
+    if (colon == NULL)
+        return -1;
+    else
+        return (colon - tagstr);
+}
+
+static int displaytext_str2tag(const char *tagstr, unsigned int *tag_len)
+{
+    int len;
+
+    *tag_len = 0;
+    len = displaytext_get_tag_len(tagstr);
+
+    if (len == -1)
+        return V_ASN1_VISIBLESTRING;
+    *tag_len = len;
+    if (len == strlen("UTF8") && strncmp(tagstr, "UTF8", len) == 0)
```

Comment:
strlen("somestring") --> sizeof("somestring")-1  Perhaps create a define for that.